test/functional/api/cas/cache_config.py

```diff
@@ -263,6 +264,110 @@ def default_nhit_params():
         return nhit_params
 
 
+# Specify layout of metadata on SSD
+class MetadataLayout(Enum):
+    striping = 0
+    sequential = 1
+    DEFAULT = striping
+
+
+# Specify how IO requests unaligned to 4KiB should be handled
+class UnalignedIo(Enum):
+    PT = 0      # use PT mode
+    cache = 1   # use current cache mode
+    DEFAULT = cache
+
+
+# Specify if IO scheduler will be used when handling IO requests
+class UseIoScheduler(Enum):
+    off = 0
+    on = 1
+    DEFAULT = on
+
+
+class KernelParameters:
+    cut_off_DEFAULT = 1
+    queue_size_DEFAULT = 65536
+    unblock_size_DEFAULT = 60000
```

Comment:
I think it will be better to make full names for those variables

Reply:
Done